### PR TITLE
Alt Text Inclusions

### PR DESCRIPTION
**Pull request by dereklh7**
Added missing alt text.

### CODE REVIEW

File: iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md [hunk 1]
```diff
@@ -50,7 +50,7 @@ The `<iisClientCertificateMappingAuthentication>` element is not available on th
 2. In **Server Manager**, click the **Manage** menu, and then click **Add Roles and Features**.
 3. In the **Add Roles and Features** wizard, click **Next**. Select the installation type and click **Next**. Select the destination server and click **Next**.
 4. On the **Server Roles** page, expand **Web Server (IIS)**, expand **Web Server**, expand **Security**, and then select **IIS Client Certificate Mapping Authentication**. Click **Next**.  
-    [![](index/_static/image2.png)](index/_static/image1.png) .
+    [![Screenshot of the I I S Client Certificate Mapping Authentication option being highlighted and selected.](index/_static/image2.png)](index/_static/image2.png) .
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md:53]: Delete (index/_static/image2.png) from inside the square brackets. Do not change the image, this should be image1

File: iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md [hunk 2]
```diff
@@ -60,7 +60,7 @@ The `<iisClientCertificateMappingAuthentication>` element is not available on th
 1. On the **Start** screen, move the pointer all the way to the lower left corner, right-click the **Start** button, and then click **Control Panel**.
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows features on or off**.
 3. Expand **Internet Information Services**, expand **World Wide Web Services**, expand **Security**, and then select **IIS Client Certificate Mapping Authentication**.  
-    [![](index/_static/image4.png)](index/_static/image3.png)
+    [![Screenshot of the I I S Client Certificate Mapping Authentication folder being highlighted and selected.](index/_static/image4.png)](index/_static/image4.png)
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md:63]: Correct the syntax like this:
Screenshot of the I I S Client Certificate Mapping Authentication folder being highlighted and selected.

Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md:63]: This is incorrect syntax, image must start with "" and do not change the image. Corrected: "![Screenshot of the I I S Client Certificate Mapping Authentication folder being highlighted and selected."

Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md:63]: This is incorrect syntax for an image. Please do not change the image.

File: iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md [hunk 3]
```diff
@@ -70,7 +70,7 @@ The `<iisClientCertificateMappingAuthentication>` element is not available on th
 2. In the **Server Manager** hierarchy pane, expand **Roles**, and then click **Web Server (IIS)**.
 3. In the **Web Server (IIS)** pane, scroll to the **Role Services** section, and then click **Add Role Services**.
 4. On the **Select Role Services** page of the **Add Role Services Wizard**, select **IIS Client Certificate Mapping Authentication**, and then click **Next**.  
-    [![](index/_static/image6.png)](index/_static/image5.png)
+    [![Screenshot of the Add Role Services Wizard with the I I S Client Certificate Mapping Authentication option being highlighted.](index/_static/image6.png)](index/_static/image6.png)
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md:73]: Please correct the syntax and do not change the image.

File: iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md [hunk 4]
```diff
@@ -79,7 +79,7 @@ The `<iisClientCertificateMappingAuthentication>` element is not available on th
 1. On the taskbar, click **Start**, and then click **Control Panel**.
 2. In **Control Panel**, click **Programs and Features**, and then click **Turn Windows Features on or off**.
 3. Expand **Internet Information Services**, then select **IIS Client Certificate Mapping Authentication**, and then click **OK**.  
-    [![](index/_static/image8.png)](index/_static/image7.png)
+    [![Screenshot of the Security folder being expanded, showing the highlighted I I S Client Certificate Mapping Authentication folder.](index/_static/image8.png)](index/_static/image8.png)
```
Comment (MeeraDi) [inline iis/configuration/system.webServer/security/authentication/iisClientCertificateMappingAuthentication/index.md:82]: Same comment as above.

**Comment by dereklh7** [comment]
@Rick-Anderson @mcleblanc Final review for added missing alt text.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by dereklh7** [comment]
@Rick-Anderson Instances of missing alt text has been added in another PR and is awaiting merge.

File: iis/publish/using-web-deploy/export-a-package-through-iis-manager.md [hunk 5]
```diff
@@ -35,36 +35,36 @@ Note: If you have not already installed the Web Deployment Tool, see [Installing
 
 1. Always make a backup. Run the following command to backup an IIS 7.0 or above server:  
 
-    [!code-console[Main](export-a-package-through-iis-manager/samples/sample1.cmd)]
+    !code-console[Main](export-a-package-through-iis-manager/samples/sample1.cmd)
```
Comment (Rick-Anderson) [inline iis/publish/using-web-deploy/export-a-package-through-iis-manager.md:38]: What's this change?

File: iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/security/customAuthorization/provider.md [hunk 6]
```diff
@@ -77,7 +77,7 @@ To support FTP authorization using a custom provider on your Web server, you mus
 4. In the **Actions** pane, click **Edit Feature settings**.
 5. In the **Authorization Feature Settings** dialog box, select **Choose a custom authorization provider** to enable FTP authorization by a custom provider. In the associated drop-down list, select a custom provider from the list.   
   
-    [![](provider/_static/image2.png)](provider/_static/image1.png)  
+    ![Screenshot of the Choose a custom authorization provider option being selected.](provider/_static/image2.png)
```
Comment (Rick-Anderson) [inline iis/configuration/system.applicationHost/sites/siteDefaults/ftpServer/security/customAuthorization/provider.md:80]: why change the image? Please review changes for mistakes like this.

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
>@Rick-Anderson Instances of missing alt text has been added in another PR and is awaiting merge.

Please don't use multiple PR's for the same file.

**Comment by dereklh7** [comment]
@Rick-Anderson All issues for this PR have been resolved. Awaiting final review.

**Review by Rick-Anderson — APPROVED** [review]

**Pull request merged**